pkg/controller/petset/identity_mappers.go

```diff
@@ -0,0 +1,202 @@
+package petset
+
+import (
+	"crypto/md5"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/api"
+	api_pod "k8s.io/kubernetes/pkg/api/pod"
```

Comment:
Don't use underscores in package names (convention) - "podapi" is pretty good here.